### PR TITLE
Small Improvements to ImageAnimatorWCS

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: sunpy/visualization/animator/image.py [hunk 1]
```diff
@@ -140,6 +138,8 @@ class ImageAnimatorWCS(ImageAnimator):
     ----------
```
Comment (dstansby) [inline sunpy/visualization/animator/image.py]: Does this not get inherited from the base class?

File: sunpy/visualization/animator/image.py [hunk 2]
```diff
@@ -160,10 +160,10 @@ class ImageAnimatorWCS(ImageAnimator):
     """
     def __init__(self, data, wcs=None, image_axes=[-1, -2], unit_x_axis=None, unit_y_axis=None,
                  axis_ranges=None, **kwargs):
-        if not isinstance(wcs, (astropy.wcs.WCS, BaseLowLevelWCS)):
-            raise ValueError("wcs data should be provided.")
+        if not isinstance(wcs, BaseLowLevelWCS):
```
Comment (dstansby) [inline sunpy/visualization/animator/image.py]: Why this change?

Comment (Cadair) [reply sunpy/visualization/animator/image.py]: `astropy.wcs.WCS` inherits `BaseLowLevelWCS` so it was redundant.

File: sunpy/visualization/animator/image.py (hunk 1 above)
Comment (Cadair) [reply sunpy/visualization/animator/image.py]: The docstring? I would assume so, but we are adding a `wcs` parameter (that was previously not included in the docstring).

**Comment by Cadair** [comment]
I need to double check this works as intended with astropy 3.1.

**Comment by Cadair** [comment]
@dstansby I checked the rendered docs and noticed more docstring issues.

**Comment by Cadair** [comment]
This does indeed work on Astropy 3.1.

File: sunpy/visualization/animator/image.py [hunk 3]
```diff
@@ -140,13 +138,15 @@ class ImageAnimatorWCS(ImageAnimator):
...
     image_axes: `list`, optional
         A list of the axes order that make up the image.
-    unit_x_axis: `astropy.units.Unit`
+    unit_x_axis: `astropy.units.Unit`, optional
         The unit of X axis.
-    unit_y_axis: `astropy.units.Unit`
+    unit_y_axis: `astropy.units.Unit`, optional
         The unit of Y axis.
-    axis_ranges: `list` of physical coordinates for the `numpy.ndarray`, optional
+    axis_ranges: `list`, optional
```
Comment (DanRyanIrish) [inline sunpy/visualization/animator/image.py]: Is there not a advantage of also giving the required type of the elements of the list?

File: sunpy/visualization/animator/image.py [hunk 4]
```diff
@@ -140,13 +138,15 @@ class ImageAnimatorWCS(ImageAnimator):
...
         Defaults to `None` and array indices will be used for all axes.
         The `list` should contain one element for each axis of the `numpy.ndarray`.
```
Comment (DanRyanIrish) [inline sunpy/visualization/animator/image.py]: Replace "`numpy.ndaray`" with "input data array".

File: sunpy/visualization/animator/image.py (hunk 3 above)
Comment (Cadair) [reply sunpy/visualization/animator/image.py]: That is in the comment for the parameter. The numpy doc format is pretty clear on what format this line should take, and a lot of sphinx themes are built around it being short.

File: sunpy/visualization/animator/image.py (hunk 4 above)
Comment (Cadair) [inline sunpy/visualization/animator/image.py]:
```suggestion
        The `list` should contain one element for each axis of the input data array.
```

**Comment by Cadair** [comment]
The coverage upload is broken, am investigating.
